lib/components/form/default-search-form.js

```diff
@@ -1,10 +1,16 @@
-import React, { Component } from 'react'
+import TriMetModeIcon from '@opentripplanner/icons/lib/trimet-mode-icon'
```

Comment:
I think we should add a new generic `ModeIcon` to otp-ui that doesn't use the TriMet-specific icons, but instead uses the ones that were originally used in this repo. One reason: we don't want all the rail icons to appear as the Max icon. Another: it is likely confusing to sprinkle in mentions of `TriMet`-branded things in this library, especially with icons which can be closely tied to agency-specific branding.

Reply:
I agree. We started off with a TriMet-focused approach... How about adding the generic icons and a generic mode-icon/leg-icon class to OTP-ui and isolate the TriMet-specific icons inside OTP-ui?

Reply:
Seems like a good idea to have a generic library specifically for otp-react-redux since we use it in more regions than just Portland, OR.

Reply:
The above topic is handled with opentripplanner/otp-ui#119 and the latest commits to this PR.